uploader.json

```diff
@@ -1611,7 +1611,7 @@
             "symbolic_name": "krb5"
         },
         {
-            "file": "/etc/krb5.conf.d/()*.*\\.conf",
+            "file": "/etc/krb5.conf.d/*",
             "pattern": [],
             "symbolic_name": "krb5_conf_d"
```

Comment:
This symbolic name should be the same name as the spec, `krb5`.  Also the existing `krb5` just above doesn't need to be a glob spec does it?  Its just a single file.